cf-java-logging-support-servlet/src/main/java/com/sap/hcp/cf/logging/servlet/filter/RequestLoggingFilter.java

```diff
@@ -35,13 +35,16 @@
     public static final String LOG_PROVIDER = "[SERVLET]";
     public static final String WRAP_RESPONSE_INIT_PARAM = "wrapResponse";
     public static final String WRAP_REQUEST_INIT_PARAM = "wrapRequest";
+    public static final String LOG_REMOTE_IP = "LogRemoteIP";
```

Comment:
What's the purpose of this? It's defined, but never used.

Reply:
The purpose is to define the key of the environment variable only once. Has been moved to the Environment class to improve clarity.